kubos/init.py

```diff
@@ -38,7 +41,12 @@ def execCommand(args, following_args):
         print >>sys.stderr, 'The project directory %s already exists. Not overwritting the current directory' % proj_name_dir
         sys.exit(1)
 
-    shutil.copytree(KUBOS_EXAMPLE_DIR, proj_name_dir, ignore=shutil.ignore_patterns('.git'))
+    #Set the correct example directory based on the desired OS
+    if args.linux is True:
```

Comment:
minor nit: we could avoid code duplication here with a simple ternary here, ala:

```python
example_dir = KUBOS_LINUX_EXAMPLE_DIR if args.linux else KUBOS_RT_EXAMPLE_DIR
shutil.copytree(example_dir, pro_name_dir, ignore=shutil.ignore_patterns('.git'))
```